components/pricing_modal/content.tsx

```diff
@@ -58,11 +61,24 @@ function Content(props: ContentProps) {
     const product = useSelector(selectSubscriptionProduct);
     const products = useSelector(selectCloudProducts);
 
+    const findProductBySku = (sku: string) => {
+        return Object.values(products || {}).find(((product) => {
+            return product.sku === sku;
+        }));
+    };
+
+    const findProductByID = (id: string) => {
+        return Object.values(products || {}).find(((product) => {
+            return product.id === id;
+        }));
+    };
+
+    const annualSubscriptionEnabled = useSelector(isAnnualSubscriptionEnabled);
+
     const isEnterprise = product?.sku === CloudProducts.ENTERPRISE;
     const isEnterpriseTrial = subscription?.is_free_trial === 'true';
-    const professionalProduct = Object.values(products || {}).find(((product) => {
-        return product.sku === CloudProducts.PROFESSIONAL;
-    }));
+    const monthlyProfessionalProduct = findProductBySku(CloudProducts.PROFESSIONAL);
```

Comment:
How do we know this is the monthly professional product? I believe both monthly and annual will have the same sku, is that correct @nickmisasi ?

Reply:
Sorry, my bad, I didn't realize they would be the same, I'll fix that.